Mathlib/Logic/Equiv/Defs.lean

```diff
@@ -858,3 +859,24 @@ def finTwoEquiv : Fin 2 ≃ Bool where
     | 0 => by simp
     | 1 => by simp
   right_inv b := by cases b <;> simp
+
+namespace Equiv
+variable {α β : Type*}
+
+/-- The left summand of `α ⊕ β` is equivalent to `α`. -/
+@[simps]
+def sumIsLeft : {x : α ⊕ β // x.isLeft} ≃ α where
```

Comment:
This might be more idiomatic as `Set.range Sum.inl ≃ α`, remembering that what you have here is actually `{x : α ⊕ β // x.isLeft = true} ≃ α`

Reply:
I am following the existing `Equiv.optionIsSomeEquiv`.

Reply:
I guess my claim is it would be reasonable to make a follow-up PR deprecating these in favor of the `range` spelling.

Reply:
I don't care enough to do so, but feel free to do it yourself 😉